Fix typo and added some links

muratcansarkalkan opened the pull request:
Hi, very nice repo. I saw a typo error and added Indeed as job board link.

sophi-li commented on README.md:
```diff
@@ -57,6 +57,7 @@ For more information on OKRs, here's [Google's guide on OKRs](https://rework.wit
 
 - Focus and accountability over three month increments.
 - OKRs are _not_ a task list. When designed right, ambitious OKRs help keep you motivated and track your progress.
+- OKRs are also a preferred business model for companies.
```
Thanks for fixing the typos! Can you remove this line? Otherwise, looks good.

muratcansarkalkan commented:
Hi, I removed that line, added that one because my previous company worked with OKRs for pay raises. Maybe that could help people be more motivated about OKRs.

sophi-li reviewed (approved):
👍
